src/utils/client.ts

```diff
@@ -82,7 +89,14 @@ export default class Client {
       .catch((e) => {
         // Ignore TypeErrors if form has redirected because `fetch` in
         // Safari will error after redirect
-        if (this.ignoreNetworkErrors && e instanceof TypeError) return;
+        if (
+          // not sure if i'm using ignoreNetworkErrors correctly - this probably should be a && so that it needs to be true for the filter to work?
+          // but the value of ignoreNetworkErrors doesn't necessarily seem correct
+          (this.ignoreNetworkErrors ||
```

Comment:
i thinks this might have to be this.ignoreNetworkErrors.current since it's a ref